src/parseTools.js

```diff
@@ -1292,6 +1319,29 @@ function sendI64Argument(low, high) {
   return low + ', ' + high;
 }
 
+// Any function called from wasm64 may have bigint args, this function takes
+// a list of variable names to convert to number.
+function from64(x) {
+  if (!MEMORY64) {
+    return '';
+  }
+  if (Array.isArray(x)) {
+    let ret = '';
+    for (e of x) ret += from64(e);
+    return ret;
+  } else {
+    return `${x} = Number(${x});`;
+  }
+}
+
+function to64(x) {
+  if (MEMORY64) {
+    return `BigInt(${x})`;
+  } else {
+    return `${x}`;
```

Comment:
Does just `return x` work?    Also maybe it should mirror the above and to `if (!MEMORY64) return x` first and omit the else?